app/views/posts/show.html.erb

```diff
@@ -1 +1,48 @@
-<h1>Here is post details</h1>
\ No newline at end of file
+<section class="my-bio bg-secondary bg-gradient">
+    <div class="container">
+            <div class="card my-4">
+                <div class="card-header post-heading">
+                    <h4># <%=@post.Title%></h4> <h4>Comments: <%=@post.comments_count%> Likes: <%=@post.likes_count%></h4>
+                </div>
+                <div class="card-body"> <p><%=@post.text%></p></div>
+                <div class="media-info">
+                    <ul>
+                        <li><a href="#">Comments: <%=@post.comments_count%></a></li>
+                        <li><a href="#">Likes: <%=@post.likes_count%></a></li>
```

Comment:
- [ ] Instead of `likes_count` it should be `likes_counter` and because of this, an error is being thrown and the method cannot be recognized.